server/graphql/v1/mutations/orders.js

```diff
@@ -558,19 +558,20 @@ export async function updateSubscription(remoteUser, args) {
     if (amount < 100 || amount % 100 !== 0) {
       throw new Error('Invalid amount');
     }
-
-    order.Subscription.deactivate();
+    await order.Subscription.deactivate();
+    order.status = status.CANCELLED;
+    await order.save();
 
     const newSubscriptionDataValues = Object.assign(omit(order.Subscription.dataValues, ['id', 'deactivatedAt']), {
       amount: amount,
       updatedAt: new Date(),
       activatedAt: new Date(),
       isActive: true,
+      status: status.PENDING,
```

Comment:
here @znarf .

Reply:
Ok, that is the subscription, and in the order?

Reply:
sorry, put on the wrong one. fixed.